ntbs-service/Pages/SearchOld/Index.cshtml

```diff
@@ -0,0 +1,72 @@
+@page

```

Comment:
https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/sort-filter-page?view=aspnetcore-3.0#add-paging Taken (and slightly adapted) from here

Reply:
I think you meant to put this comment on the search page you've added? ;)